### ISSUE TITLE
Add load() function

### ISSUE THREAD

**Issue by volcan01010** [BritishGeologicalSurvey/etlhelper#108]
### Summary

As an ETL user, I would like a simple `load` function, so that I don't have to write the INSERT query by hand.

### Description

Writing an insert query to use with `executemany` or `copy_rows` can be time consuming, and in many cases the aim is only to write the data from the input the the appropriate columns.
In this case, a `load` function could save effort by automatically generating the INSERT statement from the data given.
A simple `load` function would also make it easy to write a `copy_table_rows` function.

```python
def load(table, conn, data):
    """
    Load the data into pre-existing table on conn.
    """
    columns = get_column_names(data)
    insert_query = generate_insert_query(columns, conn)
    executemany(insert_query, conn, data)


def get_column_names(data):
    """Return the column names of a data structure."""
    # this is the tricky bit - it has to handle NamedTuple or Dictionary and "peek" into the iterable to get
    # the first item without losing it
    return column_names


def generate_insert_statement(columns, conn):
    # this function has to use a DbHelper to work out the correct placeholder for conn, then
    # write the correct statement using either named or numeric placeholders
    return insert_statement
```

**Comment by volcan01010** [comment]
A way to peek into the iterable of data/rows: https://stackoverflow.com/a/2425347/3508733

Do this before passing `first_row` into `get_column_names`.

**Comment by volcan01010** [comment]
This was solved in #109.
